utils/nctl/sh/scenarios/configs/gov96.config.toml

```diff
@@ -47,16 +47,18 @@ unit_hashes_folder = "../node-storage"
 # The duration for which incoming vertices with missing dependencies should be kept in a queue.
 pending_vertex_timeout = '1min'
 
-# If the current era's protocol state has not progressed for this long, request the latest state
-# from peers.
-standstill_timeout = '30sec'
+# If the current era's protocol state has not progressed for this long, shut down.
+# standstill_timeout = '30min'
```

Comment:
Why are commenting it out? That could in theory affect the test outcome.

Reply:
Good point: I guess it's best to keep it enabled and set to something shorter (3 minutes?) in the tests, so they don't just keep running if they go wrong.

Reply:
A good start would be to simply leave it as it was before - but we have to remember that it actually was twice that much b/c it used that timer twice before shutting down.

Reply:
You're right, there's no reason to change that behavior in this PR; I reset them.